checklist.md

```diff
@@ -121,8 +121,25 @@ Bad example:
 def get_full_name(x: str, y: str) -> str:
     return f"{x} {y}"
 ```
+5. Place each argument on a new line, including `self`, with proper indentation and formatting
+ 
+Good example:
+
+```python
+def __init__(
+        self, 
+        name: str,
+        age: int
+) -> None:
+```
+Bad example:
+
+```python
+def __init__(self, 
+             name: str,age: int) -> None:
+```
 
+6. It's good practice to specify type annotations for clarity, such as using `list[Animal]` instead of `list` to indicate a list of Animal instances
```

Comment:
```suggestion
6. It's important to use type annotations for clarity, such as `list[Animal]` instead of just `list` to specify a list of Animal instances

```